lib/membrane/rtp/vad_utils/is_speaking_estimator.ex

```diff
@@ -0,0 +1,109 @@
+defmodule Membrane.RTP.VadUtils.IsSpeakingEstimator do
+  @moduledoc """
+  Module for estimating if the user is speaking based on
+  "Dominant Speaker Identification for Multipoint Videoconferencing
+  by Ilana Volfin and Israel Cohen"
+  """
+
+  @default_parameters [
+    n1: 1,
+    n2: 10,
+    n3: 7,
+    immediate_score_threshold: 0,
+    medium_score_threshold: 20,
+    long_score_threshold: 20,
+    medium_subunit_threshold: 1,
+    long_subunit_threshold: 3
+  ]
+  @parameters Application.compile_env(
+                :membrane_rtp_plugin,
+                :vad_estimation_parameters,
+                @default_parameters
+              )
+
+  # number of levels inside one immediate interval
+  @n1 @parameters[:n1]
+  # number of immediate intervals inside one medium interval
+  @n2 @parameters[:n2]
+  # number of medium intervals inside one long interval
+  @n3 @parameters[:n3]
+
+  @target_levels_length @n1 * @n2 * @n3
+
+  @immediate_score_threshold @parameters[:immediate_score_threshold]
+  @medium_score_threshold @parameters[:medium_score_threshold]
+  @long_score_threshold @parameters[:long_score_threshold]
+
+  @medium_subunit_threshold @parameters[:medium_subunit_threshold]
+  @long_subunit_threshold @parameters[:long_subunit_threshold]
+
+  @min_activity_score 1.0e-8
+
+  @spec get_target_levels_length() :: integer
+  def get_target_levels_length(), do: @target_levels_length
+
+  @spec estimate_is_speaking(list(integer), integer) :: :speech | :silence
+  def estimate_is_speaking(levels, _level_threshold) when length(levels) < @target_levels_length,
+    do: :silence
+
+  def estimate_is_speaking(levels, level_threshold) do
+    immediates = compute_immediates(levels, level_threshold)
+    mediums = compute_mediums(immediates)
+    longs = compute_longs(mediums)
+
+    immediate_score = immediates |> hd() |> compute_activity_score(@n1, 1)
+    medium_score = mediums |> hd() |> compute_activity_score(@n2, 24)
+    long_score = longs |> hd() |> compute_activity_score(@n3, 47)
```

Comment:
Those numbers are lambda parameters in the exponential distribution. It's algorithm specific, but I think it could be added as a parameter if someone wanted to tweak it jus a bit. 

I didn't want to add it as another parameter because there is already 8 of them. I think it is a lot but can't figure out if the number can be reduced.